Cargo.toml

```diff
@@ -18,18 +18,18 @@ value = ["number-general/stream"]
 all = ["tokio-io", "value"]
 
 [dependencies]
-async-trait = "0.1"
-base64 = "0.21"
+async-trait = "0.1.78"
```

Comment:
Is there a reason why you need this specific patch version?

Reply:
see [this comment](https://github.com/DrewMcArthur/destream_json/pull/1#discussion_r1533275728) and [this comment on the specific action](https://github.com/haydnv/destream_json/pull/3#discussion_r1536009567).

since we're not running this action, these changes can be reverted, but technically a project won't compile if it requires, for example, async-trait ==v0.1.0.  i think i'll revert these changes for now and we can revisit in a future PR.